Gemfile.lock

```diff
@@ -616,7 +609,7 @@ GEM
       activerecord (>= 5.1)
       state_machines-activemodel (>= 0.8.0)
     stringex (2.8.5)
-    stripe (5.34.0)
+    stripe (5.35.0)
```

Comment:
This will conflict with https://github.com/openfoodfoundation/openfoodnetwork/pull/7862 so we'll surely need to rebase.